src/flow.h

```diff
@@ -499,6 +501,8 @@ typedef struct Flow_
 
     struct timeval startts;
 
+    const Signature *sig_drop;
```

Comment:
This isn't safe. During a rule reload rules are freed, so this will become a dangling pointer. A simpler approach would be to just store the sid and simply log that.

Reply:
I have modified the code according your advice. Can you check it?